collect_app/src/test/java/org/odk/collect/android/projects/ManualProjectCreatorDialogTest.kt

```diff
@@ -121,4 +123,13 @@ class ManualProjectCreatorDialogTest {
             Intents.release()
         }
     }
+
+    @Test
+    fun `MultiClickSafeButton should be used to protect from multiclicking`() {
```

Comment:
OK, this is very opinionated and a bit hand wavy: I honestly wouldn't bother with tests like these that just check types. It ends up feeling like the test and the implementation are exact mirrors of each other and don't really enforce behaviour. We have to trust that `MultiClickSafeButton` does the right thing for this test to really mean anything, so it doesn't really give us much confidence or protection, I'd argue.

Interested in your thoughts!

Reply:
`MultiClickSafeButton` is widely used so I think confirming it's present like in this test in enough. I thought we had tests for that button (for `MultiClickSafeImageButton` and `MultiClickSafeTextInputEditText` too) but it looks like we don't. So I would rather add tests for those views and in other places only check if they are used. I don't think it should block this issue because we have believed it worked so far (in like 10 classes) we can do that now too and create a separate issue for adding tests.

Reply:
Oh sorry, I didn't mean we needed more tests, I meant we can just delete these tests. I feel like just making the button a `MultiClickSafeButton` is enough, as we "trust" it (like you say). This is one of the few times I'm saying we can just get rid of a test rather than asking for more haha.

Reply:
Ok but then we need something more I think so I created this issue https://github.com/getodk/collect/issues/4645